package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/mongodb",
-  "version": "6.2.0",
+  "version": "6.3.0",
```

Comment:
Please do not update the package version.

Reply:
working on it. please hold.